src/plugins/analysis/linter/test/data/hello_world.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python3
-import os
+import os  # noqa: F401
```

Comment:
The noqa statements make no sense in a test file for linting.

Reply:
I added the file (or rather all test/data folders) to ruff's ignore list instead